### PR TITLE
Change Character Preferences input prompts to TGUI

### PR DESCRIPTION
**Pull request by FlameArrow57**
[UI][QoL]

## About the PR 
This PR just changes all character preference input list, number input, and text input prompts to TGUI prompts.

Examples:
age

name

image

## Why's this needed? 
TGUI

### CODE REVIEW

File: code/datums/preferences.dm [hunk 1]
```diff
@@ -500,16 +500,16 @@ datum/preferences
 				return TRUE
 
 			if ("update-age")
-				var/new_age = input(usr, "Please select type in age: 20-80", "Character Generation", src.age)  as null|num
+				var/new_age = tgui_input_number(usr, "Please select type in age: 20-80", "Character Generation", src.age, 80, 20)
 
 				if (new_age)
-					src.age = clamp(round(text2num(new_age)), 20, 80)
```
Comment (TobleroneSwordfish) [inline code/datums/preferences.dm:506]: I don't think TGUI guarantees the client won't return a wacky value, so probably best to leave the server side verification here.

File: code/datums/preferences.dm [hunk 2]
```diff
@@ -525,14 +525,14 @@ datum/preferences
 					src.pin	= null
 					return TRUE
 				else
-					var/new_pin = input(usr, "Please select a PIN between 1000 and 9999", "Character Generation", src.pin)  as null|num
+					var/new_pin = tgui_input_number(usr, "Please select a PIN between 1000 and 9999", "Character Generation", src.pin || 1000, 9999, 1000)
 					if (new_pin)
-						src.pin = clamp(round(text2num(new_pin)), 1000, 9999)
```
Comment (TobleroneSwordfish) [inline code/datums/preferences.dm:530]: ditto

File: code/procs/helpers.dm [hunk 3]
```diff
@@ -2258,11 +2258,11 @@ var/global/list/allowed_restricted_z_areas
   * Given user, will proompt user to select skin color from list (or custom) and returns skin tone after blending
   */
 /proc/get_standard_skintone(var/mob/user)
-	var/new_tone = input(user, "Please select skin color.", "Character Generation")  as null|anything in standard_skintones + list("Custom...")
+	var/new_tone = tgui_input_list(user, "Please select skin color.", "Character Generation", standard_skintones + "Custom...")
 	if (new_tone == "Custom...")
-		var/tone = input(user, "Please select skin tone level: 1-220 (1=albino, 35=caucasian, 150=black, 220='very' black)", "Skin tone picker") as null|num
-		if (!isnull(tone))
-			tone = 35 - clamp(tone, 1, 220) // range is 34 to -194
```
Comment (TobleroneSwordfish) [inline code/procs/helpers.dm:2265]: ditto

**Review by TobleroneSwordfish — CHANGES_REQUESTED** [review]

**Review by ZeWaka — CHANGES_REQUESTED** [review]
x, lgtm otherwise

File: code/datums/preferences.dm (hunk 1 above)
Comment (ZeWaka) [reply code/datums/preferences.dm:506]: TGUI specifically /can't/ guarantee this

**Review by ZeWaka — APPROVED** [review]

**Pull request merged**